### PR TITLE
chinese translation

### PR DESCRIPTION
**Pull request by hwangww**
## Description

1. Finish chinese translation and load as zh_CN;
2. Fix permission for docker/entrypoints/rails.sh;
3. Add  POSTGRES_PORT=5432 to .env example because if not config this arg,and error will be thrown by pg_isready;

**This will cause a bug**
After changing language as zh_CN,if some webhook have been configured,it will doesn't work.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

build it as an image and test it after deployed to kubernetes

## Checklist:

- [ √] My code follows the style guidelines of this project
- [√ ] I have performed a self-review of my own code
- [√ ] I have commented on my code, particularly in hard-to-understand areas
- [ √] I have made corresponding changes to the documentation
- [ √] My changes generate no new warnings
- [ √] I have added tests that prove my fix is effective or that my feature works
- [ √] New and existing unit tests pass locally with my changes
- [ √] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -12,8 +12,6 @@
 user.skip_confirmation!
 user.save!
 
-SuperAdmin.create!(email: 'john@acme.inc', password: '123456') unless Rails.env.production?
```
Comment (pranavrajs) [inline db/seeds.rb:15]: Hey @hwangww Why is this deleted?

File: config/secrets.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@
 # if you're sharing your code publicly.
 
 development:
-  secret_key_base: b89787dc4f8930ff36715598bddc8d28946c29c7d9d3c1f8272fc8eb307c4b1de6e9a20a824ac06b4a53efeaf99be20469da5355d7218851c8e87520db26a819
+  secret_key_base: 3beeee45bc938475ecba45075c53aae0f94299a83f824b25bbaf7965b4b0c60ff2b0c66c9047a026578deb5ecadabaa602891be2be66ed123a7b26876d4daddf
```
Comment (pranavrajs) [inline config/secrets.yml:14]: Hey @hwangww, why is this changed?

File: app/javascript/widget/components/template/EmailInput.vue [hunk 3]
```diff
@@ -29,6 +29,8 @@
 import { mapGetters } from 'vuex';
 import Spinner from 'shared/components/Spinner';
 import { required, email } from 'vuelidate/lib/validators';
+import { regex } from 'vuelidate/lib/validators/common';
+const phone = regex('phone', /^[1][3,4,5,6,7,8,9][0-9]{9}$/);
```
Comment (pranavrajs) [inline app/javascript/widget/components/template/EmailInput.vue:33]: I don't understand the change here.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
@hwangww Thanks for the PR, part of the changes seems good. There are a lot of unnecessary changes. Can you please remove those? I would also recommend using https://translate.chatwoot.com for translation changes.

Please see this guide. https://github.com/chatwoot/chatwoot/blob/develop/docs/contributing/translating-chatwoot-to-your-language.md

File: db/seeds.rb (hunk 1 above)
Comment (hwangww) [reply db/seeds.rb:15]: I'll check it.

**Comment by sojan-official** [comment]
@hwangww  I tried working on this PR, but there is a lot of conflicts with the upstream. I couldn't do much about them because they are translation conflicts.  Let's reopen this PR if you decide to work on this again. 

for future it will be easier if we could do translations via https://translate.chatwoot.com

**Pull request closed without merging**